### PR TITLE
Added recipe for cloudant

### PR DESCRIPTION
**Pull request by pmlandwehr**
This is an old build (`0.5.9`) intended to satisfy some other build dependencies. Once it's up we can update to the latest version. My primary concern is that according to pypi the package should also support python 3 but in my builds it failed to do so.

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Close/reopen now that we have a fixed version of `requests-futures`

**Comment by pmlandwehr** [comment]
One more try...

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

File: recipes/cloudant/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{%set name = "cloudant" %}
+{%set version = "0.5.9" %}
+{%set compress_type = "tar.gz" %}
```
Comment (ocefpaf) [inline recipes/cloudant/meta.yaml]: Too much jinja might be more trouble than it is worth.

**Pull request merged**